### DISCUSSION TITLE
render config for procurve switch

### DISCUSSION THREAD

**Discussion by wallacj** [netbox-community/netbox#17831] [Help Wanted!] [answered]
I am trying to render configs for my HP procurve \ Aruba OSS switches.  The configuration on these devices is vlan centric not interface centric.  Meaning I dont list a port and what vlans it has, but instead I list the vlan and what ports, tagged or untagged are members of that vlan.  I am having a hard time working through this in Jinja with netbox as the source.  Can anyone point me in a direction?

thanks
J

**Comment by Linkk93** [comment, marked as answer]
AOS-S switches can render the config in both ways and also accept the config both ways. you can do "show run structured" to get an interface centric config and you can go into an interface to tag VLANs. 
Maybe this helps.

But I don't know if the switches accept a config file in this structure.

**Reply by wallacj** [reply]
Thanks thats helpful, had no idea of the structured command.  i will try writing a config in structured format.
